src/style.css

```diff
@@ -1 +1,91 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 700;
+}
+
+form {
+  color: #3D4E61;
+}
+
+.submit__big {
+  margin: 20px 0;
+}
+
+form[data-qa="big"] input[data-qa="keypress"] {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  height: 70px;
+  font-size: 16px;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  padding: 0 0 0 62px;
+  border-radius: 4px;
+  font-weight: 400;
+  border: 1px solid #E1E7ED;
+  background-color: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+}
```

Comment:
Please look at how many duplicate properties you have. Please add a general class for inputs, where all these general style rules will be located and use it
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54936044/5dd3264c-18fe-4710-a804-ce18e2c18796)